app/presenters/summary/sections/employment_details.rb

```diff
@@ -27,6 +27,10 @@ def answers
         ].select(&:show?)
       end
       # rubocop:enable Metrics/MethodLength, Metrics/AbcSize
+
+      def heading
+        :income_details unless crime_application.respond_to?(:navigation_stack)
```

Comment:
What this is meant to do is to prevent this heading being displayed on the income cya page (although it's not clear whether it matters if it's displayed or not). I don't have access to any details about what the current path is so i've inferred from the crime application object (i.e whether it's an instance of the model or constructed from the adapter struct). I don't really like this though, it wonder if it's just better to display the heading anyway than do this